bora13-sp2

andreaborekova opened the pull request:
https://eso.vse.cz/~bora13/sp2/

andreaborekova committed:
bora13-sp2

nvbach91 commented on www/bora13/sp2/index.html:
```diff
@@ -0,0 +1,29 @@
+        <div id="result"></div>
+    </div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
script tagy maji byt jinde

andreaborekova committed:
Final- verzia

nvbach91 commented:
tyhle soubory maji byt ve slozce `/www/bora13/sp2/`
![image](https://user-images.githubusercontent.com/20724910/105435536-cf072d00-5c5d-11eb-80e5-1705db7adf20.png)

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+$(document).ready(() => {
+    $('#searchForm').on('submit', (e) => {
+        let searchText = $('#searchText').val();
```
elementy vybirejte predem, ne behem pouziti

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+    });
+});
+
+var apikey = "5f05d78d"
```
strednik

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+
+function getMovies(searchText) {
+
+    axios.get('http://www.omdbapi.com/?apikey=' + apikey + '&s=' + searchText)
```
pouzivejte string interpolation

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+                output += `
+            <div class="col-md-3">
+                <div class="well text-center">
+                    <img src="${movie.Poster}">
```
alt

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+                <div class="well text-center">
+                    <img src="${movie.Poster}">
+                    <h5>${movie.Title}</h5>
+                    <a onclick="movieSelected('${movie.imdbID}')" class="btn btn-primary" href="#">Movie Details</a>
```
bez inline javascript

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+}
+function movieSelected(id) {
+    sessionStorage.setItem('movieId', id);
+    window.location = 'movie.html';
```
stranka s detaily filmu vyrenderujte ve stejne strance jako vyhledavani, nepresmerujte uzivatele na jinou stranku

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+function getMovie() {
+    let movieId = sessionStorage.getItem('movieId');
+
+    var url = "http://www.omdbapi.com/?apikey=" + apikey
```
apostrof a strednik

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+
+    $.ajax({
+        method: 'GET',
+        url: url + "&i=" + movieId,
```
string interpolation

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+            let output = `
+        <div class="row">
+             <div class="col-md-4">
+              <img src="${movie.Poster}" class="thumbnail">
```
alt

nvbach91 commented on app.js:
```diff
@@ -0,0 +1,85 @@
+        </div>
+        <div class="row">
+             <div class="well">
+              <a href="index.html" class="btn btn-default">Go back</a>
```
bez presmerovani uzivatele na jinou stranku

nvbach91 reviewed (changes requested):
- pouzijte localStorage pro ukladani oblibenych filmu
- pridejte treti view pro zobrazeni oblibenych filmu + moznost jejich vymazu
- aplikace nesmi uzivatele nikam presmerovavat
- vysledky z vyhledavani nemazte, uzivatel bude chtit se vratit k puvodnimu seznamu vysledku vyhledavani
- opravte rozmery obrazku, a pouzivejte lepsi rozliseni obrazku

andreaborekova committed:
bora13-sp2

pouzijte localStorage pro ukladani oblibenych filmu=> funguje mi ulozenie filmu pomocou tlacidla SAVE, ale nemam dokonceny arraylist, a tym padom sa ulozeny film stale prepisuje
pridejte treti view pro zobrazeni oblibenych filmu + moznost jejich vymazu=> view sa zobrazi pomocou tlacidla MY FAVORITES (vymaz zatial chyba koli nefunkcnemu arraylistu)
aplikace nesmi uzivatele nikam presmerovavat=> X
vysledky z vyhledavani nemazte, uzivatel bude chtit se vratit k puvodnimu seznamu vysledku vyhledavani=>  X
opravte rozmery obrazku, a pouzivejte lepsi rozliseni obrazku => upraveny rozmer obrazkov

nvbach91 reviewed (changes requested):
- odevzdejte semestralku znovu podle navodu, semestralka musi byt ve slozce `/www/bora13/sp2`
- stale mate inline javascript a presmerovani na jinou stranku
- potrebuji abyste znovu prisla na obhajobu

andreaborekova committed:
FinalVersion-obhajoba

V aplikácii je funkcne view na ukladanie obľúbených filmov + ich vymazanie
Odstránila som presmerovanie na druhú stránku, detaily filmu sa zobrazia na pravej strane obrazovky
Aplikácia je dostupná pod odkazom:  https://eso.vse.cz/~bora13/sp2/

andreaborekova committed:
final

andreaborekova committed:
Delete app.js

andreaborekova committed:
Delete index.html

andreaborekova committed:
Delete app.js

andreaborekova committed:
Delete index.html

andreaborekova committed:
Delete style.css

andreaborekova committed:
Delete main.js

andreaborekova committed:
Delete main.css

andreaborekova committed:
Delete movie.html

andreaborekova committed:
Delete style.css

andreaborekova committed:
final_version
